Content.Shared/_RMC14/Xenonids/Construction/HiveCoreComponent.cs

```diff
@@ -34,4 +34,11 @@ public sealed partial class HiveCoreComponent : Component
 
     [DataField]
     public TimeSpan NextLesserDroneCooldown = TimeSpan.FromSeconds(125);
+
+    /// <summary>
+    /// How long a new construct can be made after the core is destroyed.
+    /// Only applies to this core's hive for xeno v xeno.
```

Comment:
This applies in general doesn't it? Hive core construction cooldown applies in distress signal too I think

Reply:
its on the hive so it wont be broken by xvx, of course still works on distress signal since xvx isnt real :trollface: